qt/scientific_interfaces/Indirect/Iqt.cpp

```diff
@@ -457,22 +463,21 @@ void Iqt::updatePropertyValues(QtProperty *prop, double val) {
              SLOT(updatePropertyValues(QtProperty *, double)));
 
   if (prop == m_properties["EHigh"]) {
-    // If the user enters a negative value for EHigh assume they did not mean to
-    // add a -
     if (val < 0) {
       val = -val;
       m_dblManager->setValue(m_properties["EHigh"], val);
     }
 
-    m_dblManager->setValue(m_properties["ELow"], -val);
+    if (m_uiForm.ckSymmetricEnergy->isChecked())
```

Comment:
We could maybe add a small unit test to check that setValue is only called on ELow when SymmetricEnergy isChecked?

Reply:
Currently this file isn't unit tested as it does not use an MVP pattern. I think it would be best to leave the unit tests for when the code has been refactored (hopefully in the near future)

Reply:
Do we have a master list of tests which need to be added and at what point they can be added?

Reply:
There is no list of required tests. All of the Indirect interfaces (excluding MSDFit, IqtFit, ConvFit and JumpFit) would ideally be re-written and tested.